client/src/TagExplorer/resource_scheme.js

```diff
@@ -273,14 +276,16 @@ export class ResourceScheme extends AbstractExplorerScheme {
     } else if (type === "set_hierarchy_scheme") {
       return { ...state, hierarchy_scheme: payload };
     } else if (type === "column_header_click") {
-      const { is_descending, sort_col } = state;
-      const clicked_col = payload;
-
+      const { sort_col } = state;
+      const clicked_col = payload.col_key;
+      const is_descending_bool = payload.is_descending_bool;
```

Comment:
Not significant, but just wanted to note that you can declare these using object destructuring, including the renaming of `col_key` to `clicked_col`. Would look like this `const { col_key: clicked_col, is_descending_bool } = payload;`